src/ConvertToSarif/ConvertOptions.cs

```diff
@@ -44,7 +44,6 @@ internal class ConvertOptions
         [Option(
             'a',
             "plugin-assembly-path",
-            Default = null,
```

Comment:
>Default [](start = 12, length = 7)

You can't provide null as a default argument. "." would work or String.Empty. CommandlineParser coughs on null. The net result of this error was that we could no longer call the converter utility without specifying this argument.